src/systems/a320_systems/src/pneumatic/wing_anti_ice.rs

```diff
@@ -411,16 +411,11 @@ impl WingAntiIceSystem {
         self.wai_valve
             .update_open_amount(&self.wai_valve_controller);
 
-        // We need both controllers to signal `on` for the
-        // system to be considered on without a fault.
-        if self.wai_valve_controller.controller_signals_on() {
-            // If a controller signals `on` while its corresponding valve is closed
-            // this means the system has a fault.
-            if self.is_wai_valve_closed() {
-                self.wai_has_fault = true;
-            } else {
-                self.wai_has_fault = false;
-            }
+        // If valve doesn't follow controller, there is a fault
```

Comment:
Just a small change:
```suggestion
        self.wai_has_fault = self.wai_valve_controller.controller_signals_on() == self.is_wai_valve_closed();
```

Reply:
True! Thank you for spotting it